composeApp/src/commonMain/kotlin/com/arkivanov/minesweeper/root/DefaultRootComponent.kt

```diff
@@ -60,7 +63,7 @@ internal fun DefaultRootComponent(componentContext: ComponentContext, storeFacto
     DefaultRootComponent(
         componentContext = componentContext,
         gameComponentFactory = { ctx, settings ->
-            DefaultGameComponent(componentContext = ctx, storeFactory = storeFactory, settings = settings)
+            DefaultGameComponent(componentContext = ctx, storeFactory = storeFactory, settings = settings, coroutineContext = SupervisorJob() + Dispatchers.Default)
```

Comment:
We need to use `Dispatchers.Main.immediate` instead of `Default` to interact with the `GameStore`. I suggest to do the following.

1. Pass `Dispatchers.Main.immediate` to `DefaultGameComponent` here.
2. Add `SupervisorJob` inside `DefaultGameComponent` when calling `coroutineContext(...)`.
3. Rename `coroutineContext` argument to `mainCoroutineContext`.

We can also put every argument on separate line here for better readbility, as the line is pretty long otherwise.